cli/root.go

```diff
@@ -63,3 +75,38 @@ var RootCommand = &cobra.Command{
 		}
 	},
 }
+
+func configureCustomPrefixes() {
+	var choreTypes, fixTypes, featureTypes []string
+	
+	if rootChorePrefixesFlag != "" {
+		choreTypes = parseCommaSeparatedPrefixes(rootChorePrefixesFlag)
+	}
+	
+	if rootFixPrefixesFlag != "" {
+		fixTypes = parseCommaSeparatedPrefixes(rootFixPrefixesFlag)
+	}
+	
+	if rootFeaturePrefixesFlag != "" {
+		featureTypes = parseCommaSeparatedPrefixes(rootFeaturePrefixesFlag)
+	}
+	
+	if len(choreTypes) > 0 || len(fixTypes) > 0 || len(featureTypes) > 0 {
+		conventionalcommits.SetCustomPrefixes(choreTypes, fixTypes, featureTypes)
+	}
```

Comment:
Can't we remove the `if` statement here, as some slices may be empty anyway, so the `SetCustomPrefixes` method has to be able to handle this.